core/src/main/java/hudson/model/Slave.java

```diff
@@ -133,10 +133,7 @@
     private /*almost final*/ DescribableList<NodeProperty<?>,NodePropertyDescriptor> nodeProperties = 
                                     new DescribableList<NodeProperty<?>,NodePropertyDescriptor>(Jenkins.getInstance().getNodesObject());
 
-    /**
-     * Lazily computed set of labels from {@link #label}.
-     */
-    private transient volatile Set<Label> labels;
```

Comment:
I have found plugins with groovy code that reference this property. I will add an exclude rule to avoid remove this property.